crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -219,12 +219,12 @@ impl Schedule {
         self
     }
 
-    /// Set whether the schedule applies buffers on final time or not. This is a catchall
+    /// Set whether the schedule applies deferred system buffers on final time or not. This is a catchall
     /// incase a system uses commands but was not explicitly ordered after a
```

Comment:
```suggestion
    /// Set whether the schedule applies deferred system buffers on final time or not. This is a catch-all
    /// in case a system uses commands but was not explicitly ordered before an instance of
```